### PR TITLE
[PB-PWA-2469]: Add localization test coverage to cypress

### PR DESCRIPTION
**Pull request by pedrochiossi**
## Description

PR to include automated localization tests in cypress coverage.

Changes:  
 - Adds some missing translations to `fr_FR.json` in `extensions/venia-sample-language-packs`
 - Changed test order to start on Default Store View instead of French since develop is set to  `STORE_VIEW_CODE=default`.

## Related Issue

Closes PWA-2469

## Acceptance

Adds cypress mock data integration test for [PWA-1415](https://jira.corp.magento.com/browse/PWA-1415)

## Verification Steps

- Check that every test step is covered in cypress automation
- Obs: test steps 12 and 13 cannot be automated due to requiring a second build with different environment variables

#### Test scenario(s) for direct fix/feature

In order
1. Install `@magento/venia-sample-language-packs` by running `yarn venia add -D @magento/venia-sample-language-packs`
2. Set language pack version to `file:../extensions/venia-sample-language-packs`
3. Set env variable `STORE_VIEW_CODE` to `default`
4. Set env variable `USE_STORE_CODE_IN_URL` to `true`
5. Run cypress localization tests against PR environment.
6. Verify that all localization tests pass
7. Verify that all steps mentioned in [PWA-1415](https://jira.corp.magento.com/browse/PWA-1415) are automated in tests (except for steps 12 and 13).
8. Run all cypress tests against PR environment.
9. Verify all cypress tests pass.

## Checklist

-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.

### CODE REVIEW

File: venia-integration-tests/src/fields/categoryPage/index.js [hunk 1]
```diff
@@ -103,3 +104,15 @@ export const filterListItemElement = '[data-cy="FilterList-item"]';
 
 export const filterDefaultCheckboxElement =
     '[data-cy="FilterDefault-checkbox"]';
+
+export const filterSidebarHeaderTitle =
+    '[data-cy="CategoryContent-root"] [data-cy="FilterSidebar-root"] [data-cy="FilterSidebar-headerTitle"]';
+
+export const productPrice =
+    '[data-cy="CategoryContent-root"] [data-cy="Gallery-root"] [data-cy="Gallery-Item-price"]';
```
Comment (DiegoBezerra89) [inline venia-integration-tests/src/fields/categoryPage/index.js:112]: camelCase pattern.

File: packages/venia-ui/lib/components/Gallery/item.js [hunk 2]
```diff
@@ -101,7 +101,7 @@ const GalleryItem = props => {
             >
                 <span>{name}</span>
             </Link>
-            <div className={classes.price}>
+            <div data-cy="Gallery-Item-price" className={classes.price}>
```
[the remark this answers is not in the document]
Comment (pedrochiossi) [reply packages/venia-ui/lib/components/Gallery/item.js:104]: ok

File: venia-integration-tests/src/fields/categoryPage/index.js (hunk 1 above)
Comment (pedrochiossi) [reply venia-integration-tests/src/fields/categoryPage/index.js:112]: ok

**Review by DiegoBezerra89 — DISMISSED** [review]

**Comment by mikhaelbois** [comment]
@pedrochiossi 
QA ⚠️ 
Other Cypress tests seems to be affected by the `data-cy` changes.
`yarn test:headless --base-url https://pr-3650.pwa-venia.com/`

File: venia-integration-tests/src/fields/accountAccess/index.js [hunk 3]
```diff
@@ -35,3 +35,27 @@ export const forgotPasswordSubmitButton =
...
+    '[data-cy="AccountMenu-root"] [data-cy="ForgotPassword-title"]';
+
+export const forgotPasswordInstructionsText =
+    '[data-cy="AccountMenu-root"] [data-cy="ForgotPassword-instructions"]';
+
+export const signInTitleText =
+    '[data-cy="AccountMenu-root"] [data-cy="SignIn-title"]';
+
+export const signInFormLabels =
+    '[data-cy="AccountMenu-root"] [data-cy="SignIn-form"] label';
```
[the remark this answers is not in the document]
Comment (pedrochiossi) [reply venia-integration-tests/src/fields/accountAccess/index.js:52]: I was trying to get all labels inside the form with one selector, I can use `data-cy` inside the form components but I'll have to create a selector for each label.

**Comment by pedrochiossi** [comment]
Closing this PR as it was merged in #3657

**Pull request closed without merging**